vsintegration/src/FSharp.Editor/Hints/HintService.fs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.Editor.Hints
+
+open Microsoft.CodeAnalysis
+open Microsoft.VisualStudio.FSharp.Editor
+open FSharp.Compiler.CodeAnalysis
+open FSharp.Compiler.Symbols
+open FSharp.Compiler.Text
+
+module HintService =
+
+    // Relatively convenient for testing
+    type NativeHint = {
+        Range: range
+        Parts: seq<TaggedText>
```

Comment:
Would prefer to have this eager, so that enumeration cannot have surprising errors or performance drawbacks.

Reply:
I am lazy so I put lazy sequences by default :D 
Sure, updated